addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -644,7 +643,7 @@
         <field name="search_view_id" ref="hr_leave_view_search_my"/>
         <field name="view_ids" eval="[(5, 0, 0),
                 (0, 0, {'view_mode': 'kanban', 'view_id': ref('hr_leave_view_kanban')})]"/>
-        <field name="domain">[('user_id', '=', uid), ('employee_company_id', 'in', allowed_company_ids)]</field>
+        <field name="domain">[('user_id', '=', uid)]</field>
```

Comment:
Shouldn't we keep a check on the company?
```suggestion
        <field name="domain">[('user_id', '=', uid), ('company_id', 'in', allowed_company_ids)]</field>
```

Reply:
No, this is now handled automatically by the new access rule so the check in the individual actions can be removed.